apps/server/src/modules/board/service/internal/board-context.service.ts

```diff
@@ -67,4 +84,17 @@ export class BoardContextService {
 
 		return usersWithRoles;
 	}
+
+	private getBoardRolesFromRoomMember(member: UserWithRoomRoles): BoardRoles[] {
```

Comment:
this could be static as it is a pure function

Reply:
To me it looks pretty unkommon to call static methods on a object instance that are not otherwise required to be static. I'm also not sure is this gives any advantage over an instance method (e.g. memory-wise)?